src/drivers/driver_zephyr.c

```diff
@@ -717,6 +717,43 @@ static int wpa_drv_zep_set_supp_port(void *priv,
 }
 
 
+static int wpa_drv_zep_signal_poll(void *priv, struct wpa_signal_info *si)
+{
+	struct zep_drv_if_ctx *if_ctx = NULL;
+	const struct zep_wpa_supp_dev_ops *dev_ops = NULL;
+	int ret = -1;
+
+	if (!priv) {
+		wpa_printf(MSG_ERROR, "%s: Invalid handle\n", __func__);
+		goto out;
+	}
+
+	if (!si) {
+		wpa_printf(MSG_ERROR, "%s: Invalid params\n", __func__);
+		goto out;
+	}
+
+	if_ctx = priv;
+	dev_ops = if_ctx->dev_ctx->config;
+
+	os_memset(si, 0, sizeof(*si));
+
+	if (dev_ops && dev_ops->signal_poll) {
+		ret = dev_ops->signal_poll(if_ctx->dev_priv, si, if_ctx->bssid);
+		if (ret) {
+			wpa_printf(MSG_ERROR, "%s: Signal polling not supported\n", __func__);
```

Comment:
wring print, this should log the failure.

Reply:
Corrected